pages/performances/index.js

```diff
@@ -84,13 +328,59 @@ export default function Performances() {
 }
 
 // Entries Table
-const EntryTable = () => {
-  return <Table columns={columns} data={data} />;
+const EntryTable = props => {
+  const columns = [
```

Comment:
Is this different from the `columns` imported on line 23?

Reply:
Yeah. The `columns` on line 23 is still the mock data which is currently used to populate the Judging table. The `columns` here on line 332 is for the Admin view table. Once the Judging table is done, we won't need the mock columns anymore.

Reply:
Right, that makes sense then. We can prob eventually move these column definitions into a separate file in the future